editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -957,6 +957,7 @@ void SpriteFramesEditor::_animation_name_edited() {
 		counter++;
 		name = new_name + "_" + itos(counter);
 	}
+	edited->set_text(0, name);
```

Comment:
If you do it unconditionally anyway, it can be moved before `while` loop and the other `set_text()` can be removed.

Reply:
Here, `name` is the new name. The case where the name has not changed has been returned early.